src/makeInfinityAdder.js

```diff
@@ -5,6 +5,22 @@
  */
 function makeInfinityAdder() {
   // write code here
+  let sum = 0;
+  const adder = (number = undefined) => {
```

Comment:
Consider using `null` instead of `undefined` as the default value for `number`. This can help avoid potential issues if `undefined` is passed explicitly as an argument.